ports/downwell/Downwell.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/downwell"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```

Comment:
Wrong ld_library export. Check the ld_library_path fix commit we had to do to fix them all.

Reply:
Sorry which is the right path?

export LD_LIBRARY_PATH="/usr/lib64:$GAMEDIR/libs"?

Reply:
I use `export LD_LIBRARY_PATH="$GAMEDIR/libs:$LD_LIBRARY_PATH"`